api/registrations/serializers.py

```diff
@@ -341,7 +342,9 @@ def get_view_only_links_count(self, obj):
         return obj.private_links.filter(is_deleted=False).count()
 
     def update(self, registration, validated_data):
-        auth = Auth(self.context['request'].user)
+        # TODO - when withdrawl is added, make sure to restrict to admin only here
```

Comment:
Super minor: typo "withdrawal"

Reply:
That is a weird way to spell that word, English Language.